src/replica/test/replica_disk_test_base.h

```diff
@@ -57,7 +57,6 @@ class replica_disk_test_base : public replica_test_base
         generate_mock_app_info();
 
         stub->_fs_manager._dir_nodes.clear();
-        stub->_fs_manager.reset_disk_stat();
```

Comment:
Is it related to this patch?

Reply:
Actually this resulted from rebasing `master` branch. In [master](https://github.com/apache/incubator-pegasus/blob/master/src/replica/test/replica_disk_test_base.h#L60) branch, this line was newly added, while it has been removed in `migrate-metrics-dev` branch. After rebasing, `migrate-metrics-dev` has this line which should have been dropped.